### ISSUE TITLE
User password encryption recovery disabled after upgrading to 8.1.1

### ISSUE THREAD

**Issue by js94x** [owncloud/core#18229]
### Steps to reproduce
1. upgrade from 8.0.4 to 8.1.1
2. followed all steps regarding encryption in documentation (Encryption / Upgrading from 8.0)
3. changed password for one user in the LDAP backend and try to set the private key password in admin area for the user
4. when i hover the password field in user table this message appears: 
Changing the password will result in data loss, because data recovery is not available for this user...

### Expected behaviour
Password reset through admins after editing password in LDAP Backend without reconfiguration.

### Actual behaviour
Users must re-enable the Password recovery in personal settings. But this doesn´t work properly, too. The message "Updating recovery keys. This may take a while" appears. And in the background my server creates directories named OC_DEFAULT_MODULE. To see if it works i started Firebug and had a look in the Network Analyzer. 

The server responds with a 503 HTTP-code ?

![2015-08-12_12h21_15](https://cloud.githubusercontent.com/assets/7836104/9223096/78ef4196-40f7-11e5-8dcd-88e3a5e1ea53.png)

Both in the web server error log and in the owncloud.log I see no errors. After changing password in LDAP backend again and try to set it as admin in Ownclouds User List it works, but after login with the user there is this message: "Private key for user is not valid! Maybe the user password was changed from outside..."

### Server configuration
**Operating system**:
Ubuntu 14.04 64x
**Web server:**
Apache 2.4.7
**Database:**
MySQL 5.5.44
**PHP version:**
5.5.9
**ownCloud version:** (see ownCloud admin page)
8.1.1 (stable)

**Updated from an older ownCloud or fresh install:**
Update...

**List of activated apps:**
Enabled:
 - activity: 2.0.1
 - encryption: 1.0.0
 - files: 1.1.10
 - files_pdfviewer: 0.7
 - files_sharing: 0.6.2
 - files_texteditor: 0.4
 - files_trashbin: 0.6.3
 - files_versions: 1.0.6
 - gallery: 0.6.0
 - updater: 0.6
 - user_ldap: 0.6.1

**Are you using encryption:** 
yes

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

LDAP (OpenLDAP)

### Client configuration
**Browser:**
Firefox 40.0

**Operating system:**
Win 7 Professional 64bit

#### Web server error log
```
[12/Aug/2015:13:40:45 +0200] "POST /index.php/apps/encryption/ajax/userSetRecovery HTTP/1.1" 200 705 "http://oc.example.org/index.php/settings/personal" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:40.0) Gecko/20100101 Firefox/40.0"
[12/Aug/2015:13:40:54 +0200] "POST /index.php/apps/encryption/ajax/userSetRecovery HTTP/1.1" 503 9393 "http://oc.example.org/index.php/settings/personal" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:40.0) Gecko/20100101 Firefox/40.0"

```

**Comment by karlitschek** [comment]
@blizzz @schiesbn What do you think?

**Comment by js94x** [comment]
There is already an update or bugfix?

**Comment by PVince81** [comment]
@schiesbn would https://github.com/owncloud/core/pull/18983 fix this ?

**Comment by schiesbn** [comment]
yes, #18983 should fix it and will be released with 8.1.4
